charts/access/ms-teams/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: teleport-plugin-ms-teams
+description: A Helm chart for the Teleport MsTeams Plugin
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: "10.1.4"
```

Comment:
```suggestion
version: "10.1.9"
```

Reply:
FYI we have already released 10.2.0.

Reply:
Only for teleport itself, right?
I'm not seeing a 10.2.0 release for plugins